src/context/WalletProvider/WalletConnectV2/components/WalletConnectDirectRow.tsx

```diff
@@ -84,9 +88,59 @@ export const WalletConnectDirectRow = () => {
     [connect],
   )
 
+  useEffect(() => {
+    const detectWallets = async () => {
+      if (!isMobile) {
```

Comment:
nitpick: preference for renaming the import to `isMobileApp` to make the intent clear (mobile app attempts detection) which gets confusing here